database_schemas/cassandra/benchflow.cql

```diff
@@ -60,6 +60,8 @@ CREATE TABLE environment_data (
 
 CREATE TABLE network_interface_data (
   network_interface_data_id uuid,
+  container_id text,
```

Comment:
@VincenzoFerme Different containers have different network usages, so just like we have resource usage for cpu, ram and IO by container, we should do the same here